roles/config-docker-machine/tasks/main.yaml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Compiled and install docker-machine from source
+  block:
+  - name: config golang
+    include_role:
+      name: config-golang
+  - name: install docker-machine from source
+    shell: |
+      set -ex
+
+      # Install bats
+      git clone https://github.com/sstephenson/bats.git '{{ ansible_user_dir }}/bats'
+      pushd '{{ ansible_user_dir }}/bats'
+      ./install.sh /usr/local
+      popd
+
+      # Install docker
+      apt-get update
+      apt-get install apt-transport-https ca-certificates
+      wget -qO- https://get.docker.com/ | sh
```

Comment:
docker installation also looks a bit unrelated with docker-machine installation,.. but OK

Reply:
Putting docker and docker-machine installation together make sense, docker-machine tests depends on it.